examples/healthData/src/main.ts

```diff
@@ -0,0 +1,51 @@
+import fs from "fs";
+import path from "path";
+import dotenv from "dotenv";
+import { createHealthDataTranslator } from "./translator";
+import { createLanguageModel, processRequests } from "typechat";
+import { HealthDataResponse } from "./healthDataSchema";
+
+// TODO: use local .env file.
+dotenv.config({ path: path.join(__dirname, "../../../.env") });
+
+const healthInstructions:string = `
```

Comment:
You shouldn't need to annotate this

```suggestion
const healthInstructions = `
```